authors/apps/articles/renderers.py

```diff
@@ -11,3 +11,12 @@ class ArticleJSONRenderer(renderers.JSONRenderer):
     def render(self, data, media_type=None, renderer_context=None):
         # Method is responsible for displaying articles
         return json.dumps({"articles": data})
+
+
+class BookmarkJSONRenderer(renderers.JSONRenderer):
+    """The BookmarkJSONRenderer returns a JSON object of bookmarks"""
+    charset = "utf-8"
+
+    def render(self, data, media_type=None, renderer_context=None):
+        # Method is responsible for displaying bookmarks
+        return json.dumps({"bookmarks": data})
```

Comment:
I think that it would be better if the message was a key in the returned response itself, so we can maintain the same standard across all endpoints, rather than always having to look for the message, like here where it would be `data['bookmarks']['message']`, instead of `data['message']`.

Reply:
resolved